app/controllers/teams/team_memberships_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Teams::TeamMembershipsController < ApplicationController
+  before_action :authenticate_user!
+
+  def destroy
+    team_membership = teams_managed_members.find(params[:id])
+
+    team_membership.destroy
+
+    redirect_to teams_team_memberships_path(team_membership.team)
+  end
+
+  private
+
+  def teams_managed_members
+    TeamMembership.where(team_id: current_user.teams_managed.pluck(:id))
```

Comment:
`select` will be more efficient than `pluck` here, as it will turn into a nested query in AR.

Reply:
Check the queries for
```
TeamMembership.where(team_id: current_user.teams_managed.pluck(:id))
```
vs
```
TeamMembership.where(team_id: current_user.teams_managed.select(:id))
```
They'll be different.
- `.pluck` returns an array
- `.select` returns an ActiveRecordRelation
In the second example above, AR will use that relationship to make a subquery. Whereas in the first example, it will pass ids in.